api/controller/EducationController.py

```diff
@@ -1,12 +1,26 @@
 from flask import Response, request
 from api.models.education import Education
+from mongoengine.errors import FieldDoesNotExist, NotUniqueError, DoesNotExist, ValidationError, InvalidQueryError
+from api.helpers.errors import EducationAlreadyExistsError
 from flask import render_template
 
 class EducationController():
     def __init__(self):
         pass
     # def get(self):
 
-    def index(self):   
+    def index(self):
         return render_template('education_documentation.html')
```

Comment:
I'm so excited that we have a landing page for our documentation.

Reply:
Same, I can't wait to finish these endpoints and flush out that documentation